### ISSUE TITLE
Add checkbox `Automatische Übersetzung für veraltete Übersetzung` for automatic translation on page form

### ISSUE THREAD

**Issue by ulliholtgrave** [digitalfabrik/integreat-cms#1875]
### Motivation

In the daily work of the municipalities the normal workflow includes updating the german content and directly wanting to translate them to other languages. Therefore, our automatic translation workflow should include a possibility to trigger automatic translations directly in the page form for now outdated translation. 

### Proposed Solution

Add another checkbox below/above (TBD) "minor_edit" that is exclusive to "minor_edit". If it is checked and the page gets updated, all translations that would be outdated and are supported by DeepL are automatically translated.

The result would be a manual created german translation and up-to-date translations in other languages.

The translation budget needs to be considered for the translation.

### Additional Context

Related to #1727 and #1663

**Comment by nikolahoff** [comment]
@ulliholtgrave if the budget for translations is not sufficient is it possible
a) to show an info: "Your budget is not sufficient. You want to translate 123 words but you only have credits for 56 words."
b) to disable the "Update" button in that case?

**Comment by ulliholtgrave** [comment]
@nikolahoff Yes, this is possible. 

However, We should consider the case when a user is trying to translate into 5 languages but his budget only allows translations for 3. In this case we would stop the translation process after 3 and inform the user about 2 missing languages due to insufficient funds. Does that sound like a reasonable solution for you?

**Comment by nikolahoff** [comment]
Sounds good. Can we show an info-box in that case?

**Comment by osmers** [comment]
@ulliholtgrave @nikolahoff how do we deal with the possible issue that Kommunen will call us and say that, had they known, they would have chosen different languages to translate the text into? Not sure how probably this is, since the budgets should be large enough. Just wanted to alert to the possibility...maybe we can stop the process before anything is translated, saying that the budget would only be large enough for x languages, and to please go to the settings and choose the ones they want?

**Comment by osmers** [comment]
@ulliholtgrave maybe @dkehne can give his input here bcs he was not a fan of our prioritising - it would be too confusing.

**Comment by timoludwig** [comment]
We're talking about a single page here, right?
I doubt that the difference between one single page being translated into language X but not into language Y makes such a huge difference.
If they really cannot live with a partially translated page, they can also manually copy the content into the DeepL UI to finish the missing languages by hand.
I doubt the programming effort is justified by such an edge case which does not have drastic implications.

**Comment by osmers** [comment]
> I doubt the programming effort is justified by such an edge case which does not have drastic implications.

Makes sense - let's keep it simple then. Might be something we need to fix in case it happens often, but I highly doubt it. In theory at least, the 50.000 words should be enough to translate Aktualisierungen.

**Comment by osmers** [comment]
@charludo maybe you can post the current design here for the checkbox and explain the idea of language selection?

**Comment by charludo** [comment]
@osmers yes, here's the current state:

1. initial situation when opening a page(/event/poi) in the editor: 
![image](https://user-images.githubusercontent.com/47758554/223482883-5520729f-34c3-447e-9e6f-31deb176f17c.png)

2. checking "automatic translation" opens this form: 
![image](https://user-images.githubusercontent.com/47758554/223483211-2927bce9-4f00-4cdc-86c7-603f9973bd3a.png)

By default, only existing translations are checked (the behaviour originally requested in this issue), but the user has the option to also create new, non-existent translations.

**Comment by osmers** [comment]
Thanks a lot - that looks very cool I think :)
